### ISSUE TITLE
socketwatcher build issue

### ISSUE THREAD

**Issue by mranney** [mranney/node_pcap#123]
Hey @zerouid I think you may have broken your socketwatcher fork. Can you take a look?

Until your changes can get pulled in upstream, it might be reasonable to just include your version with node_pcap.

```
> socketwatcher@0.2.1 install /Users/mjr/work/node-socketwatcher
> node-gyp rebuild

  CXX(target) Release/obj.target/socketwatcher/socket_watcher.o
../socket_watcher.cpp:45:1: warning: control reaches end of non-void function [-Wreturn-type]
}
^
../socket_watcher.cpp:71:38: error: no member named 'handle' in 'SocketWatcher'
  Local<Value> callback_v = watcher->handle()->Get(symbol);
                            ~~~~~~~  ^
../socket_watcher.cpp:83:28: error: no member named 'handle' in 'SocketWatcher'
  NanMakeCallback(watcher->handle(), callback, argc, argv);
                  ~~~~~~~  ^
1 warning and 2 errors generated.
```

**Comment by zerouid** [comment]
Sorry I haven't tested with node v0.10 ... now it should be fixed...

**Comment by mranney** [comment]
Fixed by #124
